app/packs/src/apps/mydb/elements/details/VersionsTableChanges.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import VersionsTableFields from 'src/apps/mydb/elements/details/VersionsTableFields';
+import VersionsTableModal from 'src/apps/mydb/elements/details/VersionsTableModal';
+import { Alert } from 'react-bootstrap';
+
+function VersionsTableChanges(props) {
+  const {
+    id, changes, handleRevert, isEdited
+  } = props;
+
+  const revertibleFields = () => {
+    if (isEdited) return -1;
+
+    let filteredFields = [];
+
+    changes.forEach(({ fields }) => {
```

Comment:
Use `map` instead of `forEach`? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map